### DISCUSSION TITLE
How can you use the pause at height and not have to babysit the print

### DISCUSSION THREAD

**Discussion by ro4short** [OrcaSlicer/OrcaSlicer#430] [Q&A] [answered]
I tried to search the quesions asked aleady but I am not seeing the information I am looking for
I am using a P1P no AMS 
I used the pause at height but I wasn't near the printer when it paused.  By the time I got back to printer the nozzle had cooled down and I was unable to pull out the filament.  Is there a way to keep the nozzle hot so I can pull out the filament and place the new color in.
TIA

**Comment by wociscz** [comment, marked as answer]
During the pause - nozzle goes to ~100C to prevent oozing, but you are free to heat it up back via the printer menu/display. 
You can also use the new "feeding" options (load/unload) which will do what you want. After loading the new filament, just go to the home screen and resume the print.

**Reply by ro4short** [reply]
Thank you!!
